app/actors/sufia/create_with_files_actor.rb

```diff
@@ -46,10 +46,32 @@ def uploaded_files
       def mark_inactive
         return true unless Flipflop.enable_mediated_deposit?
         curation_concern.state = inactive_uri
+        send_inactive_message
       end
 
       def inactive_uri
         ::RDF::URI('http://fedora.info/definitions/1/0/access/ObjState#inactive')
       end
+
+      def send_inactive_message
+        return true if receiving_users == []
+        receiving_users.each do |admin|
+          message = "#{user.name} created <a href='#{work_path}'>#{curation_concern.title.first}</a>"
+          ::User.batch_user.send_message(admin, message, 'New work to review')
+        end
+      end
+
+      # Gather all admin users (if the admin role is defined)
+      def receiving_users
+        receivers = []
+        ::User.find_each do |user|
+          receivers << user if (user.respond_to? :admin?) && user.admin?
+        end
+        receivers
+      end
+
+      def work_path
+        Rails.application.routes.url_helpers.curation_concerns_generic_work_path(curation_concern)
```

Comment:
@jcoyne Thanks, but now it's back to complaining the polymorphic_path is undefined.  This is what I'm getting for `polymorphic_path(['Rails.application.routes', curation_concern])`
*** NoMethodError Exception: undefined method `polymorphic_path' for #<Sufia::CreateWithFilesActor:0x0000000d6b1a18>

curation_concern is
`#<GenericWork id: "j3860692c", head: [], tail: [], depositor: "user18@example.com", title: ["Test title"], date_uploaded: nil, date_modified: nil, state: #<ActiveTriples::Resource:0x6b515d4(#<ActiveTriples::Resource:0x0000000d6a2ba8>)>, owner: nil, label: nil, relative_path: nil, import_url: nil, part_of: [], resource_type: [], creator: [], contributor: [], description: [], keyword: [], rights: [], rights_statement: [], publisher: [], date_created: [], subject: [], language: [], identifier: [], based_near: [], related_url: [], bibliographic_citation: [], source: [], proxy_depositor: nil, on_behalf_of: nil, arkivo_checksum: nil, access_control_id: "38080b2d-642d-4d07-987e-0690c31d5d05", representative_id: nil, thumbnail_id: nil, admin_set_id: nil, embargo_id: nil, lease_id: nil>`